src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -262,7 +262,7 @@ const getPackageDescription = async (release, arch, repoUrls, packageName) => {
   let pack;
   // if the env is stage beta then use content-sources api
   // else use image-builder api
-  if (insights.chrome.isBeta()) {
+  if (insights.chrome.isBeta() || insights.chrome.getEnvironment() === 'qa') {
```

Comment:
I would prefer to see this as a helper function instead of adjusting it everywhere. 

Otherwise looks good to me.

Reply:
Good point... the more beta features we add the bigger the chance of a footgun going off... I'll refactor on Friday when I'm back.